src/toolchain/export-android.md

```diff
@@ -0,0 +1,105 @@
+It has not been not tested without it. To be safe, it is included in the documentation for now.
+
+
+## Compiling
+
+The environment variable `CLANG_PATH` is used by bindgen's clang-sys dependency. See also
+[clang-sys documentation](https://github.com/KyleMayes/clang-sys?tab=readme-ov-file#environment-variables)
+
+Set the environment variable `CLANG_PATH` to point to Android's build of clang. Example:
+
+```bash
+export CLANG_PATH = "{androidCliDirectory}/{androidCliVersion}/ndk/{ndkVersion}/toolchains/llvm/prebuilt/{hostMachineOs}/bin/clang"
+```
+
+Then set the `CARGO_TARGET_{targetTripleHere}_LINKER` to point to the Android linker for the Android triple you are targeting.
+You need to compile your gdext library for each Android triple individually. Possible targets can be found by running:
+
+```bash
+rustup target list
+```
+
+You can find the linkers in the Android CLI directory at:
+
+```text
+"{androidCliDirectory}/{androidCliVersion}/ndk/{ndkVersion}/toolchains/llvm/prebuilt/{hostMachineOs}/bin/{targetTripleHere}{androidVersion}"
+```
+
+As of writing this, the tested tuples are:
+
+| Tuple                       | Environment Variable                            | Godot Arch     | GDExtension Config            |
+| --------------------------- | ----------------------------------------------- | -------------- | ----------------------------- |
+| `aarch64-linux-android`     | `CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER`     | `arm64`        | `android.debug.arm64`         |
+| `x86_64-linux-android`      | `CARGO_TARGET_X86_64_LINUX_ANDROID_LINKER`      | `x86_64`       | `android.debug.x86_64`        |
+| `armv7-linux-androideabi`   | `CARGO_TARGET_ARMV7_LINUX_ANDROID_LINKER`       | `arm32`        | `android.debug.armeabi-v7a`   |
+| `i686-linux-android`        | `CARGO_TARGET_I686_LINUX_ANDROID_LINKER`        | `x86_32`       | `android.debug.x86`           |
+
+Notice how the environment variables are in all caps and the triple's "-" is replaced with "_"
+
+Make sure to add all of the triples you want to support to the ``rustup`` via:
+
+```bash
+rustup target add {androidTripleHere}
+```
+
+Example:
+
+```bash
+rustup target add aarch64-linux-android
+```
+
+Putting it all together, here is an example compiling for `aarch64-linux-android`. This is also probably the most common
+Android target as of the writing of this.
+
+Assuming 2 things:
+
+1. Android CLI is installed in the `$HOME` folder.
+2. Godot is still relying on Android ndk version: 23.2.8568313. Check
+[here](https://github.com/godotengine/godot/blob/4a0160241fd0c1e874e297f6b08676cf0761e5e8/platform/android/java/app/config.gradle#L14).
+3. The downloaded Android CLI version is: 11076708_latest (update this to be the version you downloaded).
+4. This is being ran on linux. Change the `linux-x86_64` folder in `CLANG_PATH` and `CARGO_TARGET_AARCH64_LINUX_ANDROID_LINKER`
+to be your host machine's operating system.
+5. You are targeting Android version 34.
```

Comment:
Why "2 things" and then 5 points?

Also, if you really want to use numbers, use low ones like 2 as text "two". But I think "multiple", "following", etc. is typically better, because it doesn't break when someone adds/removes a bullet point.